modules/cms/twig/FrameworkNode.php

```diff
@@ -28,17 +29,34 @@ public function compile(Twig_Compiler $compiler)
 
         $compiler
             ->addDebugInfo($this)
-            ->write("echo '<script src=\"'. Request::getBasePath()
-                .'/modules/system/assets/js/framework.js\"></script>'.PHP_EOL;" . PHP_EOL)
-        ;
+            ->write("\$_minify = ".CombineAssets::class."::instance()->useMinify;" . PHP_EOL);
 
         if ($includeExtras) {
             $compiler
-                ->write("echo '<script src=\"'. Request::getBasePath()
+                ->write("if (\$_minify) {" . PHP_EOL)
```

Comment:
Why can't the ternary approach be used here too?

Reply:
It's too much for a ternary because the way I set it up, if minifying is enabled, then it loads a combined version of the minified framework + extras JS. Otherwise it loads separate files.

Reply:
Okay, thanks for the response! Can you confirm that it works well with both `{% framework %}` and `{% framework extras %}`?

Reply:
Sure. I'll post some examples.

Reply:
I could check again if you'd like, but I do remember verifying both tags with minifying and without.

Reply:
[twig filter example](https://f20bb1310757.octodock.com/)